build.sh

```diff
@@ -87,10 +87,10 @@ echo "devtools::install_git('git://github.com/rkillick/changepoint', branch = 'm
 which lsb_release > /dev/null 2>&1
 if [ $? -eq 0 ]; then
     if [ "$(lsb_release -si)" = "Ubuntu" ]; then
-        pip install --target "${PIP_TARGET_DIR}" "rpy2==2.8.5" || exit $?
+        python2.7 -m pip install --target "${PIP_TARGET_DIR}" "rpy2==2.8.5" || exit $?
```

Comment:
Is that more portable than `pip2.7` then?

Reply:
Likely, because on my system it's `pip-2.7`

Reply:
The binaries are (unhelpfully) called `pip2`, `pip3` and so on, without the minor version, on Debian. So it seems is much more portable to say `python2.7 -m pip ...`.

Reply:
OK.